test/evergreen.yml

```diff
@@ -3598,15 +3598,15 @@ tasks:
     #        Performance Long Tests           #
     ###########################################
 
-  - name: perf-test-long-btree
+  - name: perf-test-long-500m-btree-populate
     tags: ["long-perf"]
     depends_on:
       - name: compile
     commands:
       - command: timeout.update
         params:
-          exec_timeout_secs: 86400
-          timeout_secs: 86400
+          exec_timeout_secs: 21600
+          timeout_secs: 21600
```

Comment:
Guess these values are calculated by dividing the original values by 4 (since you break the original task into 4 tasks). I'm curious how much time it actually took in your patch builds.

Reply:
Each task takes almost 2 hours to finish, perhaps I can bring the number down to 3 hours, currently, it's 6. Thanks, Luke.